unit-tests/test-coldlaunch.js

```diff
@@ -112,15 +112,14 @@ describe('cold-launch', () => {
 
     // --app not provided throws an error
     it('error thrown when --app not provided', () => {
-
       let errorFunc = () => {
+
         let options = {
           'metrics': 'metrics.ldjson',
           'runs': 30
         };
         let coldlaunch = new ColdLaunch(options);
       }
-
```

Comment:
The whitespace in this one is backwards. Should be no line above the `let`, and a line after the declarations.